references/gcp-sa-auth-shared-flow/sharedflowbundle/policies/PC.CacheAccessToken.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+ Copyright 2020 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<PopulateCache name="PC.CacheAccessToken">
+    <CacheKey>
+        <Prefix>gcp.sa.access_token</Prefix>
+        <KeyFragment ref="private.gcp.service_account.client_id"/>
+    </CacheKey>
+    <Scope>Exclusive</Scope>
+    <ExpirySettings>
+        <!-- Cache for 10min -->
```

Comment:
again - if we move to named cache, we can store expiry at env level rather than hardcoding here. 

i also think you can remove the comment